src/components/datePicker/datePicker.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { DatePicker, registerDatePickerLocale } from './datePicker';
+import { useState } from 'react';
+import { ru } from 'date-fns/locale';
+
+const meta: Meta<typeof DatePicker> = {
+  title: 'DatePicker',
+  component: DatePicker,
+  parameters: {
+    layout: 'centered',
+  },
+  tags: ['autodocs'],
+  render: (args) => {
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [startDate, setStartDate] = useState<Date | undefined>(undefined);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [endDate, setEndDate] = useState<Date | undefined>(undefined);
+    // eslint-disable-next-line react-hooks/rules-of-hooks
+    const [date, setDate] = useState<Date | undefined>(undefined);
+    registerDatePickerLocale('ru', ru);
+    return (
+      <>
+        Range DatePicker:
+        <DatePicker
+          {...args}
+          value={startDate}
+          startDate={startDate}
+          endDate={endDate}
+          onChange={setStartDate}
+          selects={'start'}
+        />
+        <DatePicker
+          {...args}
+          value={endDate}
+          startDate={startDate}
+          endDate={endDate}
+          onChange={setEndDate}
+          selects={'end'}
+        />
+        Single DatePicker:
+        <DatePicker {...args} value={date} onChange={setDate} language={'ru'} />
```

Comment:
Please move a single Date picker example to a separate story and use it as a default story.
The [`button` stories](https://github.com/reportportal/ui-kit/blob/develop/src/components/button/button.stories.tsx) can be used as an example.